### PR TITLE
Additions to troubleshooting.md

### PR DESCRIPTION
**Pull request by taishi8117**
Added how to fix an issue with resolving localhost

### CODE REVIEW

File: website/content/docs/troubleshooting.md [hunk 1]
```diff
@@ -59,6 +59,43 @@ INFO: Elapsed time: 1.883s.
 
     Then fix it correspondingly.
     
+* It is also possible that your computer has an issue with resolving localhost.
```
Comment (billonahill) [inline website/content/docs/troubleshooting.md]: s/your computer/the host

File: website/content/docs/troubleshooting.md [hunk 2]
```diff
@@ -59,6 +59,43 @@ INFO: Elapsed time: 1.883s.
...
+    If the output is similar to the above, you need to modify `/etc/hosts`
+    file to correctly resolve localhost, as shown below. 
+    
+    1. Run the following command, whose output is your computer's hostname.
+    
+        ```bash
+        $ python -c "import socket; print socket.gethostname()"
+        ```
+    
+    2. Open `/etc/hosts` file as superuser and find a line containing
```
Comment (billonahill) [inline website/content/docs/troubleshooting.md]: Open the `/etc/hosts` file...

File: website/content/docs/troubleshooting.md [hunk 3]
```diff
@@ -59,6 +59,43 @@ INFO: Elapsed time: 1.883s.
...
+    
+    3. Append your hostname after the word "localhost" on the line. 
+    For example, if your hostname was `tw-heron`, then the line should
+    look like the following:
+    
+        ```bash
+        127.0.0.1   localhost   tw-heron
+        ```
+        
+    4. Save the file and restart your computer.
```
Comment (billonahill) [inline website/content/docs/troubleshooting.md]: Is rebooting really required for changing `/etc/hosts`? I typically haven't found that to be the case.

**Comment by taishi8117** [comment]
@billonahill Thanks for pointing out. It seems to have trivial conflicts with master branch now, so i'll close this, make modification and reopen the pull request

**Comment by billonahill** [comment]
@taishi8117 you typically will want to keep the same branch and PR and not close and reopen, since it makes tracking the comments, edit history, issue deps, etc. difficult. You should be able to resolve conflicts and update the branch and have it work out with the existing PR.

**Pull request closed without merging**